### DISCUSSION TITLE
Can't you prompt for keywords and functions of data sources?

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#19316] [DBeaver Development]
hello everyone,
For example, mysql keywords and functions cannot be prompted when writing sql, although they will change color after writing,

org.jkiss.dbeaver.ext.mysql.model.MySQLDialect#MYSQL_EXTRA_FUNCTIONS
![image](https://user-images.githubusercontent.com/51857567/224239887-1a5174b4-5075-4c3e-977a-fc07736786c7.png)

**Comment by LonwoLonwo** [comment]
For MySQL and MariaDB, you can enable help topics:

![2023-03-10 13_16_43-Preferences](https://user-images.githubusercontent.com/45152336/224290464-7e03dcb5-78f6-4b2b-bd6e-9fb9ff19ea95.png)

![2023-03-10 13_17_36-DBeaver Ultimate 23 0 0 - _Maria 10 1_ Script-64](https://user-images.githubusercontent.com/45152336/224290467-3af68d87-dace-4954-80a7-9004cbe83896.png)

**Reply by yangmeng66** [reply]
1 Hello, is the syntax prompt of this function in mysql prompted in the networking environment?
![image](https://user-images.githubusercontent.com/51857567/224600770-fa2b5af0-04d3-4a0b-b8b8-0afa865ed3a0.png)

2 I think this is the same as the mysql official website（ https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_substring )

3 Where is the syntax networking prompt in the MySQL code plug-in?

**Reply by LonwoLonwo** [reply]
This is the result of `select from mysql.help_topic`

**Comment by yangmeng66** [comment]
Thank you very much, you can, but for now you can only prompt keywords, like functions, stored procedures do not prompt input and return values yet... Hope to improve here in the future
